src/core/messagelogmodel.cpp

```diff
@@ -86,8 +108,16 @@ void MessageLogModel::clear()
 
 void MessageLogModel::onMessageReceived( const QString &message, const QString &tag, Qgis::MessageLevel level )
 {
-  if ( mSuppressedTags.contains( tag ) || tag == QLatin1String( "3D" ) )
-    return;
+  if ( mSuppressedModel.contains( tag ) || tag == QLatin1String( "3D" ) )
+  {
+    for ( const QString &filter : mSuppressedModel[tag] )
+    {
+      if ( message.contains( filter ) )
```

Comment:
Let's bit a bit more flexible:

```suggestion
      if ( message.contains( filter, Qt::CaseInsensitive ) )
```